appengine/helloworld-new-plugins/gradlew

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+##############################################################################
```

Comment:
Auto-generated, but needed.  This provides the gradle magic that lets you build stuff without the correct tooling on your machine.